common/src/main/scala/reactST/reactTable/SUITable.scala

```diff
@@ -145,7 +160,7 @@ class SUITable[D, Plugins, Layout](
             )
           )
         )
-      }))
+      })(^.key := "header"))
```

Comment:
Would this mean that if we have multiple tables, we'd have repeated keys? Maybe it is not a problem but what about nested tables?